### ISSUE TITLE
Regularization Techniques

### ISSUE THREAD

**Issue by twinkle485** [MLSAKIIT/stablediffusionlora#8]
Participants can apply L2 weight decay or dropout on LoRA layers. These techniques help mitigate overfitting and ensure better generalization of the model.
Ensure that you've read the guidelines present in CONTRIBUTING.md as well as the CODE_OF_CONDUCT.md.
